src/components/WebPlatformPlayer/index.js

```diff
@@ -123,7 +123,9 @@ const WebPlatformPlayer = ({ manifest, inputs }: Props) => {
     url.searchParams.set("backgroundColor", new Color(theme.colors.card).hex());
     url.searchParams.set("textColor", new Color(theme.colors.text).hex());
     url.searchParams.set("loadDate", loadDate.valueOf().toString());
-
+    if (manifest.params) {
+      url.searchParams.set("params", JSON.stringify(manifest.params));
+    }
     return url;
   }, [manifest.url, loadDate, theme, inputs]);
```

Comment:
Didn't you forget to add `manifest.params` in the memo dependencies list?

I see it has been added on the desktop side https://github.com/LedgerHQ/ledger-live-desktop/pull/4202/files#diff-d283030eec3a43257ba9fb1db3f7dd90b208284d3e005c999903d4cfdb1233c6R116

```suggestion
  }, [manifest.url, manifest.params, loadDate, theme, inputs]);
```